### PR TITLE
Fix Stargate Token List

### PR DESCRIPTION
**Pull request by Jeff-CCH**
- [x] stargate token list
- [x] test
- [x] yarn changeset

### CODE REVIEW

File: src/logics/stargate/logic.swap-token.ts [hunk 1]
```diff
@@ -10,25 +10,36 @@ import * as common from '@protocolink/common';
...
+  chainId: number;
+  tokens: common.Token[];
+}
+
+interface StargateTokenList {
+  srcToken: common.Token;
+  dstTokenLists: DstTokenList[];
+}
+
+export type SwapTokenLogicTokenList = StargateTokenList[];
```
Comment (zodahu) [inline src/logics/stargate/logic.swap-token.ts:35]: Can we simplify the code by embedding the two additional interfaces directly into SwapTokenLogicTokenList for clarity? Such as 

```
export type SwapTokenLogicTokenList = {
  srcToken: common.Token;
  dstTokenLists: {
    chainId: number;
    tokens: common.Token[];
  }[];
}[];
```

**Comment by zodahu** [comment]
LGTM

File: src/logics/stargate/logic.swap-token.ts (hunk 1 above)
Comment (Jeff-CCH) [reply src/logics/stargate/logic.swap-token.ts]: We followed the Stargate naming convention (eg. dstGasForCall), but we can change to destXXX in our logic.

**Comment by LuPoYi** [comment]
LGTM

**Pull request merged**